doc/tesseract.1.asc

```diff
@@ -203,11 +205,14 @@ following languages:
 *est* (Estonian),
 *eus* (Basque),
 *fas* (Persian),
+*fao* (Faroese),
 *fil* (Filipino),
 *fin* (Finnish),
 *fra* (French),
 *frk* (Frankish),
 *frm* (French, Middle, ca.1400-1600),
+*fry* (Western Frisian),
```

Comment:
```suggestion
*fry* (West Frisian),
```

Reply:
See https://en.wikipedia.org/wiki/West_Frisian_language.

Reply:
The reason I wrote that is the iso639 language pack on Python showing that as name:

```
$ python
Python 3.8.6 (default, Nov 26 2020, 23:33:31)
[GCC 9.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from iso639 import languages
>>> languages.get(part3='fry').name
'Western Frisian'
```

Reply:
Ah sorry, yes, obviously the English Wikipedia and ISO 639 disagree on the name. Would it help to use the ISO name, or is the Wikipedia name fine? I am not a native speaker, so I am not sure which of both variants is more common in daily life.

Reply:
I have no preference, and I also don't know what's common for the Tesseract documentation and man pages, just wanted to point out why I picked "Western". I speak Dutch but not Frysian. :-) 

I have seen the `iso639` python library return alternative names for some languages (`.names` attribute), but at least for `fry` it does not list `West Frisian` as an alternative.